.understand-anything/ONBOARDING.md

```
@@ -1,17 +1,20 @@
 # Onboarding Guide: ORISO-TenantService
 
-1. Start with `README.md`, `pom.xml`, and `src/main/java/com/vi/tenantservice/TenantServiceApplication.java`.
+1. Start with `README.md`, `pom.xml` (Java 21, Spring Boot parent 4.0.1 - the README's version claims are stale), and `src/main/java/com/vi/tenantservice/TenantServiceApplication.java`.
 2. Open `.understand-anything/README.md` and launch the dashboard using the command shown there.
 3. Follow these graph tour steps:
 
 - 1. Project Overview: `README.md`, `pom.xml`, `src/main/java/com/vi/tenantservice/TenantServiceApplication.java`
-- 2. Tenant API Boundary: `api/tenantservice.yaml`, `src/main/java/com/vi/tenantservice/api/controller/TenantController.java`
+- 2. Tenant API Boundary: `api/tenantservice.yaml`, `src/main/java/com/vi/tenantservice/api/controller/TenantController.java`, `src/main/java/com/vi/tenantservice/api/controller/TenantMediaController.java`
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)